android/app/src/main/java/labs/pooh/eaterslab/repository/dao/OccupancyStatsDao.kt

```diff
@@ -1,11 +1,15 @@
 package labs.pooh.eaterslab.repository.dao
 
 import labs.pooh.eaterslab.client.models.OccupancyStats
+import kotlin.math.min
 
 data class OccupancyStatsDao(
     var id: Int?,
     var timestamp: org.threeten.bp.OffsetDateTime,
     var occupancyRelative: Double
-)
+) {
+    val occupancyPercent: Int
+    get() = (min(1.0, occupancyRelative) * 100.0).toInt()
```

Comment:
Does it mean that server side now always return uncapped occupancy?

Reply:
Yes, it think it was the requested feature as for now we don't support roles in the API

Reply:
We *do** support roles.